modules/imap/handler_modules.php

```diff
@@ -804,7 +830,17 @@ public function process() {
             $imap = Hm_IMAP_List::connect($form['imap_server_id'], $cache);
             if (imap_authed($imap)) {
                 $this->out('imap_mailbox_page_path', $path);
-                list($total, $results) = $imap->get_mailbox_page(hex2bin($form['folder']), $sort, $rev, $filter, $offset, $limit, $keyword);
+                if (isset($this->request->get['screen_emails']) && hex2bin($form['folder']) == 'INBOX' && $this->module_is_supported("contacts")) {
+                    $contacts = $this->get('contact_store');
+                    $contact_list = $contacts->getAll();
```

Comment:
Are all existing contact emails trusted by default or only those in Trusted Senders group? I am a bit confused by the usage of trusted senders group.

Reply:
It is for all existing contact emails(in address book, trusted, etc.)